covasim/webapp/index.html

```diff
@@ -35,34 +35,36 @@
   <header>
     <b-navbar type="dark" variant="dark">
       <b-navbar-brand href="#">{{ app.title }}</b-navbar-brand>
-      <b-navbar-nav class="nav-head">
+      <b-navbar-nav class="ml-auto">
```

Comment:
On my screen, "Covasim" and "Documentation" were misaligned vertically, so I moved them to the right

Reply:
They aren't misaligned. It looks like that because you changed the font size of the .navbar-brand to be much bigger.

Reply:
They looked misaligned to me...same issue with the logo and copyright (text is too high): 
![image](https://user-images.githubusercontent.com/3239256/80427832-62f9c300-889d-11ea-82b2-acbc1efaf257.png)
I think the problem is that the _bottom_ of the text is aligned to the center of the div, instead of the center of the text being aligned to the center of the div. With the top bar, it's more complicated, because ideally the small text should have its center align with the center of the "m" in "Covasim" (not with the div height or with the center of the "C").